### ISSUE TITLE
def repackage_hidden(h) function occurred error.

### ISSUE THREAD

**Issue by o-matsuo** [kanoh-k/WikiNLP#1]
when i use this function, function worked.

-------------------------------------------------------
def repackage_hidden(h):
    """Wraps hidden states in new Tensors, to detach them from their history."""
    if isinstance(h, torch.Tensor):
        return h.detach()
    else:
        return tuple(repackage_hidden(v) for v in h)
-------------------------------------------------------

please see.
https://discuss.pytorch.org/t/help-clarifying-repackage-hidden-in-word-language-model/226/7
